src/screens/QRScannerScreen.js

```diff
@@ -69,24 +69,27 @@ export default function QRScannerScreen() {
           <Background />
           <CameraDimmer cameraVisible={cameraVisible}>
             {initializeCamera && (
-              <QRCodeScanner
-                cameraDim={cameraDim}
-                contentPositionTop={HeaderHeight}
-                dsRef={dsRef}
-                enableCamera={cameraVisible}
-              />
+              <View>
+                <ScannerHeader>
+                  <BackButton
+                    color={colors.whiteLabel}
+                    direction="left"
+                    onPress={handlePressBackButton}
+                    testID="goToBalancesFromScanner"
+                  />
+                  <EmulatorPasteUriButton />
+                </ScannerHeader>
+                <QRCodeScanner
+                  cameraDim={cameraDim}
+                  contentPositionTop={300}
+                  dsRef={dsRef}
+                  enableCamera={cameraVisible}
+                />
+              </View>
             )}
           </CameraDimmer>
+
           {android ? <DiscoverSheet ref={dsRef} /> : null}
-          <ScannerHeader>
```

Comment:
Does this mean if the camera is not initialized there's no header?
I'm not 100% sure what the initializeCamera flag does but I'd just make sure there's still a header even if you reject camera permissions

Reply:
nice catch, works find on android was messy on ios.

Fixed now:

ios: http://cloud.skylarbarrera.com/ios-disco.mov

android: http://cloud.skylarbarrera.com/Screen-Recording-2021-03-17-19-07-52.mp4